L1Trigger/L1THGCal/python/hgcalTriggerPrimitiveDigiProducer_cfi.py

```diff
@@ -18,19 +18,16 @@
                      tdcnBits = digiparam.hgceeDigitizer.digiCfg.feCfg.tdcNbits,
                      tdcOnsetfC = digiparam.hgceeDigitizer.digiCfg.feCfg.tdcOnset_fC,
     
-
-                 )
+)
     
 be_parameters = cms.PSet(
-    linLSB = cms.double(100./1024.),
-    triggerCellTruncationBits = cms.uint32(0),
+    linLSB = cms.double( (100./1024)*(2**(0)) ),
```

Comment:
* Use the parameters defined in  fe_codec
* Use a different name, like `cellLSB`. There is no linearization done here.